pkg/synthetictests/alerts.go

```diff
@@ -8,13 +8,36 @@ import (
 
 	"k8s.io/client-go/rest"
 
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
 	"github.com/openshift/origin/pkg/monitor/monitorapi"
 	"github.com/openshift/origin/pkg/synthetictests/allowedalerts"
 )
 
 func testAlerts(events monitorapi.Intervals, restConfig *rest.Config, duration time.Duration, recordedResource *monitorapi.ResourcesMap) []*junitapi.JUnitTestCase {
+
+	kubeClient, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return []*junitapi.JUnitTestCase{
+			{
+				Name: "Alert setup, kube client",
```

Comment:
This results in an "only failed" test.  it needs to set to "pass" in the majority of cases so that aggregation works properly.  I'm surprised we don't have clients we can pass in here.